oxidation/crates/parsec_core_fs/migrations/2022-04-26-103106_chunks/up.sql

```diff
@@ -0,0 +1,10 @@
+-- Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
```

Comment:
date in the filename is wrong, you should have a look at which Parsec version introduced the change in the datamodel and use it release date instead

Reply:
Is it important ?

Reply:
not having a date is valid, having a date is valid. However having an incorrect date is dangerous since we may end up using it by mistake (typically if all the subsequent migrations have a correct date we will forget this one is a special snowflake)

So I would say it is better not to have date at all (since we use a number prefix in the backend postgresql migrations it would be better to be consistent), but is this possible with diesel ?

Reply:
Of course it is possible ^^